Setup/UpgradeSchema.php

```diff
@@ -38,6 +38,22 @@ public function upgrade(SchemaSetupInterface $setup,
             }
         }
 
+        if (version_compare($context->getVersion(), 'beta-3.3.4', '>')) {
+            $tableName = $setup->getTable('sales_order');
+            if ($setup->getConnection()->isTableExists($tableName) == true) {
+                $setup->getConnection()->addColumn(
+                    $tableName,
+                    'rzp_order_id',
+                    [
+                        'nullable' => true,
+                        'type'     => \Magento\Framework\DB\Ddl\Table::TYPE_TEXT,
+                        'length'   => 255,
```

Comment:
@enishant , can we make length to just 55 here.

Reply:
@ramth05 updated length to 55